components/src/input/tex/extensions/color-v2/build.json

```diff
@@ -0,0 +1,5 @@
+{
+    "component": "input/tex/extensions/color-v2",
```

Comment:
We have `config_macros` and `tag_format` packages. So I would like to apply the same naming convention here: `color_v2`.